project/accounts/templates/accounts/profile.html

```diff
@@ -15,4 +15,5 @@ <h1>{% translate "User profile" %}</h1>
             {% translate "Save" %}
         </button>
     </form>
+    <a href="{%url 'password_change' %}">Change Password</a>
```

Comment:
Let's switch to using hyphens instead of underscores and add localization support.
```suggestion
    <a href="{%url 'password-change' %}">
       {% translate "Change Password" %}
    </a>
```